styles/program.css

```diff
@@ -0,0 +1,116 @@
+.program {
+  background: url(../assets/program/pattern.png), #282b32;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+}
+
+.programSectionTitle {
+  height: 100px;
+  display: flex;
+  flex-direction: column;
+  color: white;
+  font-size: 36px;
+  font-weight: bold;
+  padding-top: 60px;
+}
+
+.music {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-around;
+  background-color: rgba(80, 79, 79, 0.5);
+  height: 100px;
+  width: 90%;
+  min-width: 355px;
+  max-width: 605px;
+  gap: 15px;
+  border-radius: 4px;
+  border: 4px solid transparent;
+}
```

Comment:
- [ ] Nice work making the cards responsive :+1: However it would be nice to also add the program image to the card on the mobile version to follow the provided design.
### Your design :point_down: 
![Screenshot from 2023-06-29 02-08-35](https://github.com/IsmaelMastronardi/Capstone1/assets/91142432/57d67276-c9e4-49dc-b074-07e062b60ef7)
### What it is supposed to look like :point_down: 
![Screenshot from 2023-06-29 02-07-07](https://github.com/IsmaelMastronardi/Capstone1/assets/91142432/8ed36147-c9a8-4421-bf7f-575b3eccc7f2)